index.html

```diff
@@ -0,0 +1,89 @@
+    <span id="sign-in"><a href="https://health.smedanregister.ng/auth/login/" target="_blank" style="text-decoration: none; color: rgb(0 0 0);" >Sign in</a></span>
+    <button id="start-button"> <a href="https://health.smedanregister.ng/signup/" target="_blank" style="text-decoration: none; color: white"> Get Started </a><img src="forward arrow icon.png" width="12px" height="10px"> </button>
+    
+  </header>
+  <div class="main-body">
+    <h1><b>Health Insurance</b></h1>
+      <p>Pay nothing when you visit the hospital.<br>
+      Here's how: subscribe for N120,000 annual cover with a payment of
+      <br>N1,100/month.</p>
+      <button id="second-button"> <a href="https://health.smedanregister.ng/signup/" target="_blank" style="text-decoration: none; color: white;"> Get cassava health for businesses </a><img src="forward arrow icon.png" width="12px" height="10px"> </button>
+      <p style="color:red">Powered by <img src="cassava_image-removebg-preview (1).png" width="180px" height="50px"> </p>
+  </div> 
+
+  <div class="main-body2">
+    <p>
+      <h2 id="h2">Casava Health Insurance?</h2>
+      Our Health Insurance provides you access to affordable healthcare at a hospital close to you. In <br> case of illness, visit the hospital, Casava will take care of your medical bills.</p>
+  </div>
+
+  <div id="main-body3">
+    <div><h2>Subscribe</h2><br>
+    Subscribe and protect your <br>health today. Your first month is <br> free!</div>
+    <div><h2>Access Healthcare</h2><br> Choose your preferred hospital from a <br>wide hospital network, close to your <br> location.</div>
+    <div><h2>Telemedicine</h2><br>Get free access to virtual <br> consultation with over 200 <br> licensed doctors.</div>
+  </div>
+ 
+  
+  <div id="mainbody4">
+    <div class="mainbody4-1"> <img src="wepik-2022421-135956.jpg" width="700px"; height="450px"> 
+    </div>
+
+    <div class="mainbody4-2">
+      <div>
+        <h2 id="price">As low as<br>
+      N1,100 / month</h2>
+      Casava Health Insurance has been designed to<br>
+      cover the most important health needs at rock<br>
+      bottom prices. <br><br>
+      <button class="button-price"> <a href="https://health.smedanregister.ng/#benefits" target="_blank" style="text-decoration: none; color: white"> See What's Covered <span class="covered"></a><img src="forward arrow icon.png" width="12px" height="10px"></span> </button>
+      </div>
+    </div>
+
+    <div class="mainbody4-3">
+      <div>
+        Get healthcare in<br> over 95 hospitals in<br> Lagos.
+        <br><br><button class="button-price"> <a href="https://health.smedanregister.ng/#benefits" target="_blank" style="text-decoration: none; color: white"> See all Hospitals <span class="covered"></a><img src="forward arrow icon.png" width="12px" height="10px"></span> </button>
+      </div>
+    </div>
+
+    <div class="mainbody4-4"><br><img src="happy-good-looking-man.png" >
+    </div>
+  </div>
```

Comment:
Hey there, you've really done well regarding this project. However, i will suggest you employ the HTML best practices by using HTML Semantics  (e.g Section, footer, nav-bar etc). Because it will be easier for any developer to read your code